eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -83,6 +84,13 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.executeUpdate("CREATE INDEX channel_updates_timestamp_idx ON audit.channel_updates(timestamp)")
       }
 
+      def migration89(statement: Statement): Unit = {
+        statement.executeUpdate("CREATE TABLE audit.path_finding_metrics (amount_msat BIGINT NOT NULL, fees_msat BIGINT NOT NULL, success BOOLEAN NOT NULL, duration INTERVAL NOT NULL, timestamp TIMESTAMP WITH TIME ZONE NOT NULL, experiment_name TEXT NOT NULL)")
```

Comment:
Using the `interval` data type was a bad suggestion, sorry. For example both queries below return `0`:

```sql
select extract('millisecond' from interval '60 seconds')
select date_part('milliseconds', interval '60 seconds')
```